#!/usr/bin/env python
PACKAGE = "controlPublisher"
# The following line should not be here for Groovy and above
import roslib;roslib.load_manifest(PACKAGE)

# For Groovy and above, the following line should be
# from dynamic_reconfigure.parameter_generator_catkin import *
from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("thruster1", double_t, 0, "ratio of thruster 1", 1, 0, 1)
gen.add("thruster2", double_t, 0, "ratio of thruster 2", 1, 0, 1)
gen.add("thruster3", double_t, 0, "ratio of thruster 3", 1, 0, 1)
gen.add("thruster4", double_t, 0, "ratio of thruster 4", 1, 0, 1)
gen.add("thruster5", double_t, 0, "ratio of thruster 5", 1, 0, 1)
gen.add("motor_test_mode",bool_t, 0 ,"for testing motors", False)

exit(gen.generate(PACKAGE, "controlPublisher", "thrusterRatios"))

